pack_unpack.py

```diff
@@ -0,0 +1,32 @@
+#Script for unpacking/packing a pptx file
+#Sample run - python pack_unpack.py 1 Desktop/sample.pptx Desktop/unpack
+import zipfile
+import argparse
+import os
+import shutil
+
+parser = argparse.ArgumentParser(description='Script to unpack/pack pptx file----->1-unpack, 2-pack')
+parser.add_argument('option', help='Unpack/Pack Mode, 1-unpack')
+parser.add_argument('pptx_path', help='Path to original pptx file')
+parser.add_argument('unpack_path', help='Path to destination folder for unpacked pptx file')
+args = parser.parse_args()
+
```

Comment:
would you mind inserting some default parameters, to suit the contents of this repo?

Reply:
There cannot be any default parameters for "pptx_path" and "unpack_path". We can set the first  parameter "option"'s default value to be 1 (unpack).

Reply:
Hmm, a few lines later you extract the values of two path arguments.  If the argument is null, it can be replaced with the values for our repo. Right?